src/test/kfs_rest_parser_test.cpp

```diff
@@ -812,6 +812,22 @@ TEST_F(KFSRestParserTest, parseValidRequestWithNoDataButBinaryInputsParameter) {
     ASSERT_EQ(parameter->second.int64_param(), 4);
 }
 
+TEST_F(KFSRestParserTest, parseStringInput) {
+    std::string request = R"({
```

Comment:
DO we have rest test with real (non-empty data)?

Reply:
No, there are no high level unit tests (?) for http message -> string handling. There were no tests for 1d u8 and it stays like that in this PR. Do you see it neccessary?

Reply:
@bstrzele

Reply:
At least 1 positive end to end